db/migrate/20170403135631_set_visible_to_trued_for_custom_attributes.rb

```diff
@@ -0,0 +1,9 @@
+class SetVisibleToTruedForCustomAttributes < ActiveRecord::Migration[5.0]
+  def up
+    CustomAttribute.where(:visible => nil).update_all(:visible => true)
```

Comment:
We are adding the column so all rows would be `nil`.  I do not see a need for `where(:visible => nil)`?

I realize someone could rollback to this migration without removing the column, but that seems highly unlikely and in my opinion more of a reason to include this in the migration that adds the column.

@lpichler What is the need to have this in a separate migration?